samples/net/http_server/README.rst

```diff
@@ -0,0 +1,151 @@
+.. _http_server:
+
+HTTP server
+###########
+
+The HTTP server sample demonstrates how to host a HTTP server on a nRF70 Series device.
+
+.. contents::
+   :local:
+   :depth: 2
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Overview
```

Comment:
I would add a sentence listing the supported transport options
- Wi-Fi
- LTE (future)
and the fact that this is using the connection manager by default.

Reply:
Will refurbish the docs a bit after I've tested with cellular